CHANGELOG.md

```diff
@@ -10,8 +10,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Bugfixes
 
+* Added missing mapping for azure/metrics input. [#1474](https://github.com/elastic/package-registry/pull/1474)
```

Comment:
Added as a bugfix this PR https://github.com/elastic/package-registry/pull/1474
Is that ok?